docs/manifest.md

```diff
@@ -2,16 +2,13 @@
 
 TODO(John Sirois): Finish writing docs: https://github.com/a-scie/lift/issues/32
 
+% TODO(John Sirois): https://github.com/a-scie/lift/issues/32
+%  Flip `allow_missing_doc` to "no" and work through errors until the manifest format is fully 
+%  documented.
 ```{dataclass} science.model:Application
----
-toctree_maxdepth: 2
-allow_raw_typenames: "no"
-
-# TODO(John Sirois): https://github.com/a-scie/lift/issues/32
-#  Flip to "no" and work through errors until the manifest format is fully documented.
-allow_missing_doc: "yes"
-
-link_tables: "yes"
-recurse_tables: "yes"
----
+:toctree_maxdepth: 2
```

Comment:
The Myst update warns about the deprecated directive option syntax and comment syntax (and our doc build treats warnings as errors); so this updates to the approved syntax for both options directives and comments.